conans/test/integration/tools/system/package_manager.py

```diff
@@ -0,0 +1,198 @@
+import mock
```

Comment:
File must be named ``test_package_manager.py`` otherwise it doesn't run in CI?

Reply:
🤦